src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
You have multiple divs with the class 'stars' and 'stars--#'. Semantic HTML tags can be used here to improve the readability and SEO of your page. How about using a <section> tag for each 'stars' div instead of a <div> tag? It would be more semantic and meaningful.